app/Controllers/Http/UsersController.ts

```diff
@@ -0,0 +1,76 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Profile from 'App/Models/Profile'
+import User from 'App/Models/User'
+import Hash from '@ioc:Adonis/Core/Hash'
+import CreateUser from 'App/Validators/CreateUserValidator'
+
+
+export default class UsersController {
+    public async getAllUsers() {
+        return User.all()
+    }
```

Comment:
This is not needed and as it's not mentioned in the task.

Reply:
Done